app/utils/IncidentP5Helper.scala

```diff
@@ -118,61 +119,19 @@ class IncidentP5Helper(
       ).flatten
     )
 
-  def endorsementDateRow: Option[SummaryListRow] =
-    buildRowFromAnswer[XMLGregorianCalendar](
-      answer = data.Endorsement.map(_.date),
-      formatAnswer = formatAsDate,
-      prefix = "departure.notification.incident.index.endorsement",
-      id = None,
-      call = None
-    )
-
-  def authorityRow: Option[SummaryListRow] =
-    buildRowFromAnswer[String](
-      answer = data.Endorsement.map(_.authority),
-      formatAnswer = formatAsText,
-      prefix = "departure.notification.incident.index.authority",
-      id = None,
-      call = None
-    )
-
-  def endorsementCountryRow: Future[Option[SummaryListRow]] =
-    data.Endorsement
-      .map {
-        endorsementType =>
-          refDataService.getCountry(endorsementType.country) map {
-            countryResponse =>
-              val countryToDisplay = countryResponse.fold[String](identity, _.description)
-              buildRowFromAnswer[String](
-                answer = Some(countryToDisplay),
-                formatAnswer = formatAsText,
-                prefix = "departure.notification.incident.index.endorsementCountry",
-                id = Some(s"country-$displayIndex"),
-                call = None
-              )
-          }
-      }
-      .getOrElse(Future.successful(None))
-
-  def locationRow: Option[SummaryListRow] =
-    buildRowFromAnswer[String](
-      answer = data.Endorsement.map(_.place),
-      formatAnswer = formatAsText,
-      prefix = "departure.notification.incident.index.location",
-      id = None,
-      call = None
-    )
-
   def endorsementSection: Future[StaticSection] =
     for {
-      endorsementCountryRow <- endorsementCountryRow
+      endorsementCountryRow <- data.Endorsement match {
```

Comment:
Map over the endorsement. If the endorsement exists, instantiate the endorsement helper. Return type will likely be a Future[Option[StaticSection]]